src/bin/cql-stress-cassandra-stress/operation/mod.rs

```diff
@@ -16,6 +23,121 @@ use scylla::{
 };
 pub use write::WriteOperationFactory;
 
+use crate::settings::CassandraStressSettings;
+use crate::stats::ShardedStats;
+
+use self::row_generator::RowGenerator;
+
+/// A specific CassandraStress operation.
+#[trait_variant::make(CassandraStressOperation: Send)]
+pub trait LocalCassandraStressOperation: Sync + Send + Sized {
+    type Factory: CassandraStressOperationFactory<Operation = Self>;
+
+    const DEFAULT_TABLE_NAME: &'static str = "standard1";
+    const DEFAULT_COUNTER_TABLE_NAME: &'static str = "counter1";
+
+    fn get_stressed_table_name() -> &'static str;
```

Comment:
At a glance, it looks like this method is called only from within other methods from the same implementation of the trait. Perhaps we can drop it?

Reply:
Actually, this function is only used in implementations of `CassandraStressOperationFactory` during creation of `PreparedStatement`.

It is mainly needed to differentiate between `read` and `counter_read` operations (see `read.rs`). One of them reads from `standard1` table while the other reads from `counter1` table.

Reply:
In that case, there is a bug in `read.rs`, in `GenericReadOperationFactory::new`, it uses a hardcoded name of `"standard1"` to create the prepared statement.

Still, I'm not sure I like the idea to put this into `CassandraStressOperation`. It's only really used in the case you mentioned; although there are actually two other cases in `counter_write.rs` and `write.rs`, I don't think they count because they spell the implementing type directly and we don't really utilize the fact that this is a trait method there.